### PR TITLE
Starter Template.

### PR DESCRIPTION
**Pull request by oke-aditya**
Does the following.

1. Initializes Empty Documentation using mkdocs.
2. `.git` files, templates.
3. `.github` for `ISSUE_TEMPLATE` and for `actions` later on (I'm thinking to greet new contributor etc no CI CD here lets keep it beginner friendly)
4. We need to fill up the `CONTRIBUTING.md` and `README.md` still, I just created them.
5. `data` folder is from where users can make use of `datasets` if they are small enough else simply create link to dataset.

6. We need to think how main folder of the repo should be. How do we subidivide it?

Last but not the least Thanks for the opportunity 😃 

P.S. Please don't add `hacktoberfest-approved` label to my PRs 🙂 .

### CODE REVIEW

**Review by GokuMohandas — APPROVED** [review]
LGTM 👍 and for the main doc, perhaps organizing by application area (can have nested structure) will be best?

**Comment by oke-aditya** [comment]
I'm thinking the same. Let's discuss it in an issue.
Something like
```
src (name better)
====> Machine Learning
====> Deep Learning
=======> DL-CV
=======> DL-NLP
====> Reinforcement Learning
====> Computer Vision
====> NLP


```

**Comment by oke-aditya** [comment]
P.S. Please `Squash and Merge` all PRs. Keeps git history clean 👍 and ensures 1 PR = 1 contribution.

**Comment by oke-aditya** [comment]
Adds `greetings.yml` To greet a new user.

Adds `stale.yml` to mark stale issues.

I have requested for `review-notebook-app`. 
This will make code reviews for Jupyter Notebook very simple since Github does not render the diffs properly :(

**Comment by oke-aditya** [comment]
Can we Squash Merge this?
I don't want to merge my own PR 😄

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```
Comment (ayulockin) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: ..of what to *be* expected ..

**Review by ayulockin — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (oke-aditya) [reply .github/ISSUE_TEMPLATE/bug_report.md:22]: I will leave all documentation updates in next PR 👍

**Pull request merged**